www/src/components/CodeBlock.tsx

```diff
@@ -37,7 +37,7 @@ const {
 
 export type CollapsibleLiveEditorTypes = {
   children: React.ReactNode;
-  clickToCopy: (arg: string) => void;
```

Comment:
@adamstankiewicz do you know why `arg: string` was in here before?

Reply:
Most likely, we previously passed some kind of parameter here, but now this parameter does not exist.
Good catch! 💯